src/main/java/edu/cornell/kfs/sys/businessobject/PaymentSourceWireTransferExtendedAttribute.java

```diff
@@ -0,0 +1,119 @@
+package edu.cornell.kfs.sys.businessobject;
+
+import org.kuali.kfs.krad.bo.PersistableBusinessObjectBase;
+import org.kuali.kfs.krad.bo.PersistableBusinessObjectExtension;
+
+public class PaymentSourceWireTransferExtendedAttribute extends PersistableBusinessObjectBase implements PersistableBusinessObjectExtension {
+    
+    private String documentNumber;
+    private String payeeAccountTypeCode;
+    private String bankStreetAddress;
+    private String bankProvince;
+    private String bankSWIFTCode;
```

Comment:
The old equivalent of this extended attribute had "...SWIFTCode" for the regular bank's property but had "...SwiftCode" for the correspondent bank's property, and that discrepancy was carried over to this new extended attribute. If it's not too much work, it would be good to update those properties to be consistent about the "SWIFT" capitalization.

Reply:
fixed